site/docs/guides/basic-drawing.md

```diff
@@ -240,17 +262,34 @@ void pixel(int x, int y) {
 ```go
 func pixel (x int, y int) {
     // The byte index into the framebuffer that contains (x, y)
-    var idx = (y*160 + x) >> 2;
+    var idx = (y*160 + x) >> 2
+
+    // Calculate the bits within the byte that corresponds to our position
+    var shift = uint8((x & 0b11) << 1)
+    var mask = uint8(0b11 << shift)
+
+    // Use the first DRAW_COLOR as the pixel color
+    var color = uint8(*w4.DRAW_COLORS & 0b11)
+
+    // Write to the framebuffer
+    w4.FRAMEBUFFER[idx] = (color << shift) | (w4.FRAMEBUFFER[idx] & ^mask)
+}
+```
+
+```odin
+pixel :: proc "c" (x int, y int) {
+    // The byte index into the framebuffer that contains (x, y)
+    idx := (y*160 + x) >> 2
 
     // Calculate the bits within the byte that corresponds to our position
-    var shift = uint8((x & 0b11) << 1);
-    var mask = uint8(0b11 << shift);
+    shift := u8((x & 0b11) << 1)
+    mask := u8(0b11 << shift)
 
     // Use the first DRAW_COLOR as the pixel color
-    var color = uint8(*w4.DRAW_COLORS & 0b11);
+    color = u8(*w4.DRAW_COLORS & 0b11)
```

Comment:
Odin's dereference syntax is `w4.DRAW_COLORS^`

Reply:
Yeah, my mistake. I overlooked it.